static/js/pages/individual-contributions.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var $ = require('jquery');
+
+var tables = require('../modules/tables');
+var helpers = require('../modules/helpers');
+var columns = require('../modules/columns');
+
+var donationTemplate = require('../../templates/receipts.hbs');
+
+$(document).ready(function() {
```

Comment:
How many other `$(document).ready()` calls are made in the app, and are they (potentially) made multiple times on the same page?  If so, that can become problematic because there's no guarantee of the order in which they're run.  Oftentimes it's better to create objects that contain the functionality and provide an `init` method on them that takes care of what is being done in the `$(document).ready()` function(s).  With these `init` methods you can have one `$(document).ready()` to manage them in a central place that you are then in control of.

I searched through the code base and it doesn't appear like it's an issue if I understand the structure correctly, where most of these `$(document).ready()` functions are intended for a single page and wouldn't conflict with one another, but I did note one in `init.js` and `modules/toggle.js` as well so I wanted to call it out just in case.

We can review this in more detail if you'd like, I'm happy to talk about the concern if it's not clear and/or help investigate further to see if it is something we need to consider.